README.md

```diff
@@ -55,7 +55,7 @@ the JSON file's path.
 + `icon-size` (number, optional) - Size of all the icons inside the DMG
 + `window` (object, optional) - Window options
     + `position` (object, optional) - Position when opened
-        + `x` (number, required) - X position relative to bottom of the screen
+        + `x` (number, required) - X position relative to right of the screen
```

Comment:
Oh, so the origin is in the bottom left, right? I'll amend my commit in a couple of hours.